build/cmake/DefinePlatformSpecifc.cmake

```diff
@@ -97,21 +97,14 @@ elseif(WITH_STDTHREADS)
 endif()
 
 # C++ Language Level
-set(CXX_LANGUAGE_LEVEL "C++${CMAKE_CXX_STANDARD}")
```

Comment:
I believe CXX_LANGUAGE_LEVEL is used in the build summary output which is why it was being built as a string.